R/kiss-report.R

```diff
@@ -4,47 +4,57 @@
 #'
 #' This is a constructor function.
 #'
-#' @param url An HREF to the report in KM as described by
-#'   \link{http://support.kissmetrics.com/api-update.html}
+#' @param productId The product id to run the report against
+#' @param segment - a KissSegment specifying the segment to run the report
+#'   against.
+#' @param calculations - a list of KissCalculations representing the columns of
```

Comment:
Highlight that any calculations with type matching /_date_/ will be treated as a timestamp and converted to POSIXct. Everything else will be returned as a character string.